play/wikirecent-dbt/README.md

```diff
@@ -54,15 +68,23 @@ To show off that capability, we're going to create [materialized views](https://
 on top of streaming Wikipedia data using dbt.
 
 1. To start, let's set up a stream of Wikipedia's recent changes, and simply write all the data we see
-   to a file. From a new shell, run:
+   to a file.
+
+   > If (and only if!) you are running Materialize via a Docker container, run `docker exec -it [materialized container id] /bin/sh` before `curl`-ing to create this file directly on your container.
```

Comment:
Minor nit: Perhaps instead of "directly on your container", you could say "directly within the materailized container"?